commands/src/main/java/net/apartium/cocoabeans/commands/parsers/WithParserFactory.java

```diff
@@ -36,35 +37,81 @@ public class WithParserFactory implements ParserFactory {
 
 
         try {
-            Constructor<? extends ArgumentParser<?>>[] ctors = (Constructor<? extends ArgumentParser<?>>[]) withParser.value().getDeclaredConstructors();
-            return List.of(new ParserResult(newInstance((Constructor<ArgumentParser<?>>[]) ctors, withParser.priority()), obj instanceof Method ? Scope.VARIANT : Scope.CLASS));
+            Constructor<? extends ArgumentParser<?>>[] ctors = (Constructor<? extends ArgumentParser<?>>[]) withParser.value().getConstructors();
+
+            ArgumentParser<?> argumentParser = newInstance(
+                    (Constructor<ArgumentParser<?>>[]) ctors,
+                    withParser.priority(),
+                    withParser.keyword()
+            );
+
+            if (argumentParser == null)
+                return List.of();
```

Comment:
....

Reply:
Yeah what about it?
I don't return null just empty list